src/python/turicreate/toolkits/sound_classifier/sound_classifier.py

```diff
@@ -289,7 +273,7 @@ def create(dataset, target, feature, max_iterations=10,
 
     state = {
         '_class_label_to_id': class_label_to_id,
-        '_custom_classifier': custom_NN,
+        '_custom_classifier': sc_model,
```

Comment:
I would prefer to keep the variable name `custom_NN`. I think that's a more descriptive name. It also matches language used in the user guide.

Reply:
We refer to our Sequential model as `custom_NN` in the MXNet class so I didn't want to make it confusing. Any thoughts @nickjong ?

Reply:
I think `custom_NN` probably is the clearer name, since this object only contains the neural network classifier, not the audio preprocessing and feature extraction that are also part of the overall Sound Classifier model